### ISSUE TITLE
Folder is not stored offline

### ISSUE THREAD

**Issue by Id3aFly** [nextcloud/desktop#4110]
### Expected behaviour
All folders should be stored offline (at my PC where Nextcloud Desktop is running)

### Actual behaviour
Not all folders are stored locally (at my PC where Nextcloud Desktop is running)
Apparently Nextcloud was looking for changes in remote folders, which are even not synced locally:
![grafik](https://user-images.githubusercontent.com/5712180/147210052-f8f4e2f3-9cc8-433a-aac2-8f3f2b1065e0.png)
![grafik](https://user-images.githubusercontent.com/5712180/147210168-fb5ba3e9-9d36-4421-bd7a-bf551af35c0b.png)
(All folders and system files are visible, nothing is hidden)

### Steps to reproduce
This is a good question. I just detected it and have no idea for what reason this folder is not synced. I remember I had this issue also like ~1 year ago and commented it somewhere here, but I cannot find it anymore. So I guess it has rather less to do with the client / os as with the syncing algorithm.

### Client configuration
Client version: 3.4.1
Operating system: Win 11
OS language: German
Installation path of client: Default (C:\Program Files\Nextcloud)

### Server configuration
Nextcloud version:
NextcloudPi with:
![grafik](https://user-images.githubusercontent.com/5712180/147209251-deface06-86e0-4626-a71c-dc3c6843cca1.png)

I will provide logs, if required. Are logs anonymized? Or do they contain sensitive information?

**Comment by allexzander** [comment]
I am not sure I follow what the problem is. Could you rephrase/elaborate/provide examples?

**Comment by Id3aFly** [comment]
Same as the last time I cannot directly provide more really helpful information. 

It is a folder that I had there for like 1 year. I don't know when it stopped syncing.
However, I have something in mind that I once deleted that folder locally? But I'm not sure about that. So maybe it didn't delete online and it came to this inconsistency.
I can share of course any log data you need.

Or is there something like a debugging mode so I could focus on that folder?

**Comment by mgallien** [comment]
Please provide the desktop client logs:
- Since 3.1: Under the "General" settings, you can click on "Create Debug Archive ..." to pick the location of where the desktop client will export the logs and the database to a zip file.
- On previous releases: Via the command line: nextcloud --logdebug --logwindow or nextcloud --logdebug --logfile log.txt (See also https://docs.nextcloud.com/desktop/3.1/troubleshooting.html#log-files)
We can also sends you a link to upload the files without sharing them on github.
Thanks!

**Comment by Id3aFly** [comment]
Yes, please share me a link to upload the logs privately or is there a way to share anonymized information?
